examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -1228,6 +1241,7 @@ def compute_time_ids(original_size, crops_coords_top_left):
         pipeline = StableDiffusionXLPipeline.from_pretrained(
             args.pretrained_model_name_or_path, vae=vae, revision=args.revision, torch_dtype=weight_dtype
         )
+        pipeline.scheduler = noise_scheduler
```

Comment:
I would keep the default noise scheduler and instead initialize it have the additional arguments actually if that's okay. Here's an example:

https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth_lora.py#L1362C1-L1372C114

Here, instead we would do:

```python
pipeline.scheduler = pipeline.scheduler.from_config(pipeline.scheduler.config, **scheduler_args)
```

Reply:
reusing the noise scheduler from training also didn't work here.